--- D:/Repos/readium-shared-js-kivuto-github/build-output/_single-bundle/readium-shared-js_all.js	Wed Sep 06 15:36:42 2017
+++ D:/Repos/readium-shared-js-kivuto-github/build-output/_single-bundle/readium-shared-js_all_patched.js	Wed Sep 06 15:27:28 2017
@@ -2736,6 +2736,14 @@
 	// Try to shortcut find operations (as opposed to filters) in HTML documents
 	if ( !seed ) {
 
+        // KIVUTO: Modified version of jQuery for issue in IE11 (Kaizen-ReaderWindowsDesktop 123b38ae)
+        try {
+            document === document; //may cause permission denied
+        }
+        catch (err) {
+            document = window.document; //resets document, and no more permission denied errors.
+        }
+
 		if ( ( context ? context.ownerDocument || context : preferredDoc ) !== document ) {
 			setDocument( context );
 		}
@@ -7633,8 +7641,14 @@
 		if ( !view || !view.opener ) {
 			view = window;
 		}
+        //Bug 11024 in windows 7, ie11 windows.getComputedStyle sometime crash, redefine it to avoid the issue.
+        try {
+		  return view.getComputedStyle( elem );
+        }
+        catch (ex) {
+            return document.getComputedStyle(elem, null);
+        }
 
-		return view.getComputedStyle( elem );
 	};
 
 var swap = function( elem, options, callback, args ) {
